.github/workflows/push-mimir-build-image.yml

```diff
@@ -2,7 +2,7 @@ name: Build and Push mimir-build-image
 
 # configure trigger by pull request
 on:
-  pull_request:
+  pull_request_target:
```

Comment:
Which concrete problem does `pull_request_target` solve versus `pull_request`?

Reply:
the thread i copied on the description https://github.com/dependabot/dependabot-core/issues/3253, claim pull_request_target would grant dependabot write permission. 🪄

Reply:
Aha, will look.

Reply:
I think we shouldn't do this, because as far as I can understand, this would just be a hack to allow PR builds access to secrets, not just Dependabot. We should instead configure Dependabot secrets explicitly, as we've started doing.

Reply:
this was already the case 2 weeks ago. the aim is to fix the pipeline between the time we implement github app for all workflows. It will take sometime